.circleci/config.yml

```diff
@@ -49,6 +49,8 @@ jobs:
           command: yarn install
       - run:
           working_directory: ~/opencti/opencti-platform/opencti-graphql
+          environment:
+            CONNECTORS_MANIFEST_REF: stable
```

Comment:
This will not guaranty that we will get the right manifest version, do you think it's possible to use the version (${CIRCLE_TAG} maybe) instead ? Better to fail release than do an inconsistent release IMO.

Reply:
I removed it to target only master for now